doc/install_basic.txt

```diff
@@ -0,0 +1,47 @@
+.. _install_basic:
+
+Basic user install instructions
+===============================
+
+These are the 3 simple ways of obtaining Theano, and additional requirements
+will change depending on the way you choose.
+
+pip
+~~~
+    
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. Use the [sudo] or [--user] option depending on your admin rights.
+
+.. code-block:: bash
+
+    [sudo] pip install Theano [--user]
+
+conda
+~~~~~
+
+say something about conda and miniconda
+
+Anaconda
+~~~~~~~~
+
+If you don't want to bother with anything, Theano can be acquired through
+Anaconda_.
+
+
+.. _pip: http://pypi.python.org/pypi/pip
+.. _anaconda: https://docs.continuum.io/
+
+.. _testing_installation:
+
+Testing your installation
```

Comment:
You mean the testing and/or anaconda? I know we spoke about having a test page all by itself, but a page only for that line of code seems the extreme opposite from going where we are now with everything on one page to every details on separate pages. Extremes are usually not good! Plus the test is specific to what kind of installation you would pick.